odps/readers.py

```diff
@@ -360,6 +360,8 @@ def __next__(self):
         values = self._readline()
         if values is None:
             raise StopIteration
+        if len(values) == 0:
```

Comment:
Is it possible to merge this condition with the previous one?

Reply:
You mean like

```python
if values is None or len(values) == 0:
    raise StopIteration
```

the `len` part will throw a TypeError for objects that are truly NoneType

Reply:
This will not happen if `values is None` as Python will not evaluate the second expression.

Reply:
You are right @wjsi . Sorry for my mistake. Indeed it is possible to merge the condition to the previous one.

Reply:
I have updated the code to reflect the changes in this discussion.